src/verify.c

```diff
@@ -210,6 +228,31 @@ static bool parse_options(int argc, char **argv)
 		case 'Y':
 			cmdline_option_picky = true;
 			break;
+		case 'X':
```

Comment:
I intentionally left out updating the man page, because --picky is also not documented and I didn't want to conflict with whatever someone else might be doing right now to fix that issue #335.

But I can add documentation for --picky and --extra-picky here, once we agree on what the actual options should be.

Do you want me to remove --picky entirely or add --picky-tree instead of the current --extra-picky?

Reply:
I was suggesting something like using '--picky' to enable the 'scan everything rather than just the files listed in the manifests', and '--picky-tree=/usr' and '--picky-tree=/' as options to be the 'restrict the effects of --picky to just parts of the tree'. Then what is currently '--picky' would be '--picky', and the new '--extra-picky' would be '--picky --picky-tree=/'.

Reply:
I like @icarus-sparry's idea if it still works with @pohly's use-case.

Reply:
Yes, sounds good.

Reply:
I've implemented --picky-tree as proposed and updated the docs.